src/index.html

```diff
@@ -4,9 +4,396 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo" href="#">
+        <img src="./images/logo.png" alt="moyo logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#laptops-computers"
+            >Laptops & Computers</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#vidoe">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="card" data-qa="card">
+        <div class="card__image"><img src="images/imac.jpeg" alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"></div>
+
+        <div class="card__title">
+          <p class="card__title-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+
+          <p class="card__title-product-code">
+            Product code:
+            <span>195434</span>
+          </p>
+        </div>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__review-text">
+            Reviews:
+            <span>5</span>
+          </p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-name">Price:</p>
+
+          <p class="card__price-amount">$2,199</p>
+        </div>
+
+        <button class="button" data-qa="card-hover">buy</button>
+      </div>
+
+      <div class="card">
+        <div class="card__image"><img src="images/imac.jpeg" alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"></div>
```

Comment:
```suggestion
        <div class="card__image">
          <img src="images/imac.jpeg" alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)">
        </div>
```
Don't put a nested element in the one line with the parent element. Fix in all places